ui/src/i18n/locale-it.json

```diff
@@ -288,6 +288,53 @@
       "others": "Altri",
       "office_hours_start": "Inizio orario lavorativo",
       "office_hours_end": "Fine orario lavorativo"
+    },
+    "tags": {
+        "dashboard": "dashboard",
+        "summary": "sommario",
+        "agent": "agente",
+        "session": "sessione",
+        "caller": "chiamante",
+        "call": "chiamata",
+        "lost call": "chiamate perse",
+        "performance": "performance",
+        "hourly": "oraria",
+        "geographic": "geografica",
+        "load": "carico",
+        "hour": "ora",
+        "area": "area",
+        "queue": "coda",
+        "position": "posizione",
+        "avg": "avg",
+        "average": "media",
+        "duration": "durata",
+        "wait": "attesa",
+        "calls": "chiamate",
+        "answer": "risposte",
+        "lost": "perse",
+        "done": "effettuate",
+        "ivr": "ivr",
+        "choices": "scelte",
+        "unique": "unico",
+        "total": "totali",
+        "good": "risposte",
+        "failed": "fallite",
+        "timeout": "timeout",
+        "exitempty": "exitempty",
+        "exitkey": "exitkey",
```

Comment:
I think these should be translated, I have now idea how because I do not understand the meaning.

Reply:
@andrea-marchionni can you help us?